cv/index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Wanacode - cv</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,400italic,300italic,300,500italic,700,700italic,900,900italic'
+        rel='stylesheet' type='text/css'>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+<div class="wrapper">
+  <aside class="sidebar-wrapper">
+    <div class="profile-container">
+      <img class="profile" src="img/avatar.png" alt=""/>
+      <h1 class="name">John Doe</h1>
+      <h3 class="tagline">Front end Developer</h3>
+    </div>
+
+    <ul class="side-list">
+      <li><a href="mailto: yourname@email.com">john.doe@site.com</a></li>
+      <li><a href="tel:0123 456 789">234 234 549</a></li>
+      <li><a href="#" target="_blank">site.com</a></li>
+      <li><a href="#" target="_blank">linkedin.com/in/johndoe</a></li>
+      <li><a href="#" target="_blank">github.com/johndoe</a></li>
+    </ul>
+
+    <h2 class="container-block-title">Interests</h2>
+    <ul class="side-list">
+      <li>Gameing</li>
+      <li>Football</li>
+      <li>Cooking</li>
+    </ul>
+  </aside>
+
+  <main class="main-wrapper">
+    <article class="section">
```

Comment:
section is not a descriptive class name. Please use something more descriptive so it is easier to figure out what is going on when another programmer is reading your code. It is a section describing career, so maybe `career-section`.

Reply:
How about using `<section>` semantig tag